app/javascript/dashboard/components/widgets/conversation/components/SLACardLabel.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div
+    v-if="showSLACard"
```

Comment:
Top level v-if's are not a good idea. The component should never decide if they will toggle themselves, best is to let the parent handle this. That way we won't have to pass that additional prop `showSLACard`